TypeScript: OwnerInfo type missing avatar_url property causes error in analyze-user page

## Problem

`apps/web/components/Analyze/utils.ts` defines the `OwnerInfo` interface without an `avatar_url` property, but `apps/web/app/analyze-user/[login]/page.tsx` tries to access `data.avatar_url`.

**Files:**
- `apps/web/components/Analyze/utils.ts` line 13 (interface definition)
- `apps/web/app/analyze-user/[login]/page.tsx` line 32 (usage)

**Error:**
```
app/analyze-user/[login]/page.tsx(32,25): error TS2339: Property 'avatar_url' does not exist on type 'OwnerInfo'.
```

**Current interface:**
```typescript
export interface OwnerInfo {
  type: 'User' | 'Organization' | 'Bot';
  login: string;
  name: string;
  id: number;
  bio: string;
  public_repos: number;
  // Missing: avatar_url
}
```

**Usage causing error:**
```typescript
<PersonJsonLd
  name={displayName}
  login={data.login}
  bio={data.bio}
  avatarUrl={data.avatar_url}  // Error: Property does not exist
/>
```

## Impact

- TypeScript compilation fails with error TS2339
- May cause runtime issues if the API response includes avatar_url but the type definition does not account for it
- The PersonJsonLd component expects an avatarUrl prop, which is currently not type-safe

## Suggested Fix

Add `avatar_url` to the `OwnerInfo` interface:

```typescript
export interface OwnerInfo {
  type: 'User' | 'Organization' | 'Bot';
  login: string;
  name: string;
  id: number;
  bio: string;
  public_repos: number;
  avatar_url?: string;  // Add this line (optional since it may not always be present)
}
```

--- a/apps/web/components/Analyze/utils.ts
+++ b/apps/web/components/Analyze/utils.ts
@@ -17,6 +17,7 @@ export interface OwnerInfo {
   id: number;
   bio: string;
   public_repos: number;
+  avatar_url: string;
 }
 
 export const getOwnerInfo = (owner: string): Promise<OwnerInfo> => {

